randovania/gui/preset_settings/patcher_energy_tab.py

```diff
@@ -56,6 +60,17 @@ def __init__(self, editor: PresetEditor, game_description: GameDescription, wind
             self.progressive_damage_reduction_check.setVisible(False)
             self.heated_damage_box.setVisible(False)
 
+        # In-Game Difficulty
+
+        if self.game_enum == RandovaniaGame.METROID_PRIME:
+            self.ingame_difficulty_combo.setItemData(0, IngameDifficulty.NORMAL)
+            self.ingame_difficulty_combo.setItemData(1, IngameDifficulty.HARD)
+            self.ingame_difficulty_combo.setItemData(2, IngameDifficulty.EITHER)
```

Comment:
we may have a helper function already to populate a combo from an enum. if not, you can do:
```py
for i, difficulty in enumerate(IngameDifficulty):
    self.ingame_difficulty_combo.setItemData(i, difficulty)
```